### PR TITLE
[BUG] fix random state overwrite in MiniRocketMultivariate

### PR DESCRIPTION
**Pull request by fkiraly**
Correctly resetting estimators in https://github.com/alan-turing-institute/sktime/pull/2562 unmasked a hidden bug with `MiniRocketMultivariate`, which was overwriting its `random_state` variable in `__init__`. This has been fixed.

### CODE REVIEW

**Comment by ltsaprounis** [comment]
Not sure if this should be fixed in the context of this PR or it's a separate issue, BUT:

Setting the numpy seed as done in line [140](https://github.com/alan-turing-institute/sktime/blob/2ab70b78caf30f3a5fa1ea516ba4179dd44ba632/sktime/transformations/panel/rocket/_minirocket_multivariate.py#L140) is considered bad practice: 
https://numpy.org/doc/stable/reference/random/generated/numpy.random.seed.html?highlight=seed#numpy.random.seed

We should change the randomnesss control to something like this: 

```python
from sklearn.utils import check_random_state

...

rng = check_random_state(self.random_state)

# replace all calls from np.random to rng
rng.choice(...)
rng.randint(...)
```

**Comment by fkiraly** [comment]
I´d say out of scope, but kindly report as an issue.

In-scope for this PR is "fixing non-compliant interface", I don´t know much about multirocket.

**Review by ltsaprounis — APPROVED** [review]
Approved as a temp fix of the overwriting behaviour. In the future, randomness handling should be changed as described in #2568.
